data/settings/icon-grid-vi.json.in

```diff
@@ -73,11 +67,7 @@
     "org.blender.Blender.desktop"
   ],
   "eos-folder-social.directory": [
-    "eos-get-us.zoom.Zoom.desktop",
-    "eos-link-twitter.desktop",
-    "evolution.desktop",
-    "eos-skype.desktop",
-    "eos-link-gmail.desktop"
+    "evolution.desktop"
```

Comment:
Well, I feel like a Social folder containing one app would rather be just the app, but let's leave revamping the desktop to a separate task.

Reply:
Thought the same and agree we should handle updating the grid defaults as a separate task.

Reply:
https://phabricator.endlessm.com/T31629